code/modules/mob/living/living.dm

```diff
@@ -51,10 +51,16 @@
 	return ..()
 
 /mob/living/proc/ZImpactDamage(turf/T, levels)
+	Knockdown(levels * 50)
+	if(ishuman(src))
```

Comment:
![image](https://user-images.githubusercontent.com/20987591/80020477-670e9500-84d9-11ea-9775-365b9429ad9f.png)

Reply:
all memeing aside, override this proc on /mob/living/carbon/human/ZImpactDamage(turf/T, levels) instead of doing a type check.

Reply:
Please, Floyd, review and constructive criticism is appreciated, but try not to discourage learning contributors.